lib/EasyPost/Service/EventService.php

```diff
@@ -28,4 +31,35 @@ public function all($params = null)
     {
         return self::allResources(self::serviceModelClassName(self::class), $params);
     }
+
+    /**
+     * Retrieve all payloads for an event.
+     *
+     * @param string $id The event id
+     * @return mixed
+     */
+    public function retrieveAllPayloads($id)
+    {
+        $url = $this->instanceUrl(self::serviceModelClassName(self::class), $id) . '/payloads';
+
+        $response = Requestor::request($this->client, 'get', $url);
+
+        return InternalUtil::convertToEasyPostObject($this->client, $response);
+    }
+
+    /**
+     * Retrieve a payload for an event.
+     *
+     * @param string $id The event id
+     * @param string $payloadId The payload id
+     * @return mixed
+     */
+    public function retrievePayload($id, $payloadId)
+    {
+        $url = $this->instanceUrl(self::serviceModelClassName(self::class), $id) . '/payloads' . '/' . $payloadId;
```

Comment:
No need to split two strings up when they can be consolidated

```suggestion
        $url = $this->instanceUrl(self::serviceModelClassName(self::class), $id) . '/payloads/' . $payloadId;
```